README.md

```diff
@@ -279,19 +291,25 @@ with DAG(
 
 ## Unit tests
 
-From the root project directory: `python -m unittest discover -s tests/unit`
+From the root project directory: `make unit`
 
 ## Integration tests
 
 Integration tests require access to ClickHouse server. Tests use connection
     URI defined [via environment variable][airflow-conn-env]
     `AIRFLOW_CONN_CLICKHOUSE_DEFAULT` with `clickhouse://localhost` as default.
 
-Run from the project root: `python -m unittest discover -s tests/integration` 
+You can start clickhouse server locally with the following command:
+
+```bash
+make run-clickhouse
+```
+
+And then run from the project root: `make integration`
```

Comment:
How will this work? Will you attach to a docker-run container and run the command inside? So far I expect that `run-clickhouse` runs a docker container and `integration` still executes the commands locally. Am I wrong?